dataprofiler/tests/profilers/test_numeric_stats_mixin_profile.py

```diff
@@ -502,3 +502,30 @@ def test_merge_num_zeros_and_negatives(self):
         num_profiler._add_helper(other1, other2)
         self.assertEqual(num_profiler.num_zeros, 0)
         self.assertEqual(num_profiler.num_negatives, 0)
+
+    def test_diff_helper(self):
+        """
+        Checks _diff_helper() works appropriately.
+        """
+        other1, other2 = TestColumn(), TestColumn()
+        other1.min = 3
+        other1.max = 4
+        other1._biased_variance = 1
+        other1.sum = 6
+        other1.match_count = 10
+        
+        other2.min = 3
+        other2.max = None
+        other2._biased_variance = 9
+        other2.sum = 6
+        other2.match_count = 20
+        
+        expected_diff = {
+            'min': 'unchanged',
+            'max': [4, None],
+            'sum': 'unchanged',
+            'mean': 0.3,
+            'variance': -8.362573099415204,
+            'stddev': -2.0238425028660023
```

Comment:
where do we get mean, var and std from other1 and other2? Should we only calculate the difference based on the given stats from other1 and other2?

Reply:
I've been just subtracting the difference from other1 and other2. I thought about recalculating it based on the intermediate statistics, but then I thought what would be the point? Like, I think a data scientist wants to know how much variance has shifted, but I don't think recalculating it particularly makes sense. I'm open to ideas tho